mesonbuild/compilers.py

```diff
@@ -1786,6 +1789,9 @@ def get_options(self):
     def get_option_link_args(self, options):
         return options['c_winlibs'].value[:]
 
+    def get_linker_allow_undefined(self):
+        return ['/FORCE']
```

Comment:
According to the `/FORCE` docs, we should be using `/FORCE:UNRESOLVED` here to be exact: https://msdn.microsoft.com/en-us/library/70abkas3.aspx